### PR TITLE
docs(zh-cn): update  github-pages.md

### PR DESCRIPTION
**Pull request by qiudaoermu**
## Check List

**Please read and check followings before submitting a PR.**

- [ ] I want to publish my theme on Hexo official website.
  - [ ] I have read the [theme publishing doc](https://hexo.io/docs/themes#Publishing).
  - [ ] `name` is unique.
  - [ ] `link` URL is correct.
  - [ ] `preview` URL is correct.
  - [ ] `preview` URL web site is rendered correctly.
  - [ ] Add a screenshot to `source/themes/screenshots`.
  - [ ] Screenshot filename is same as value of `name`.
  - [ ] Screenshot size is `800 * 500`.
  - [ ] Screenshot file format is `png`.
- [ ] I want to publish my plugin on Hexo official website.
  - [ ] I have read the [plugin publishing doc](https://hexo.io/docs/plugins#Publishing).
  - [ ] `name` is unique.
  - [ ] `link` URL is correct.
- [ ] Others (Update, fix, translation, etc...)
  - Languages:
  - [ ] `en` English
  - [ ] `ko` Korean
  - [ ] `pt-br` Brazilian Portuguese
  - [ ] `ru` Russian
  - [ ] `th` Thai
  - [x] `zh-cn` simplified Chinese
  - [ ] `zh-tw` traditional Chinese

### CODE REVIEW

**Comment by qiudaoermu** [comment]
zh-ch docs keep up with  zh-tw;

File: source/zh-cn/docs/github-pages.md [hunk 1]
```diff
@@ -1,68 +1,100 @@
 ---
-title: 将 Hexo 部署到 GitHub Pages
+title: 在 GitHub Pages 上部署 Hexo
 ---
 
-在本教程中，我们将会使用 [Travis CI](https://travis-ci.com/) 将 Hexo 博客部署到 GitHub Pages 上。Travis CI 对于开源 repository 是免费的，但是这意味着你的站点文件将会是公开的。如果你希望你的站点文件不被公开，请直接前往本文 [私有 Repository](#私有 Repository) 部分。
+本文将使用 [GitHub Actions](https://docs.github.com/en/actions) 部属至 GitHub Pages，此方法适用于公开或私人储存库。若你不希望将整个资料夹推上 GitHub，请参阅 [一键部属](#一键部属)。
```
Comment (SukkaW) [inline source/zh-cn/docs/github-pages.md:5]:
```suggestion
本文将使用 [GitHub Actions](https://docs.github.com/en/actions) 部署至 GitHub Pages，此方法适用于公开或私人储存库。若你不希望将整个资料夹推上 GitHub，请参阅 [一键部署](#一键部署)。
```

File: source/zh-cn/docs/github-pages.md [hunk 2]
```diff
@@ -1,68 +1,100 @@
...
-如果你更希望你的站点部署在 `<你的 GitHub 用户名>.github.io` 的子目录中，你的 repository 需要直接命名为子目录的名字，这样你的站点可以通过 `https://<你的 GitHub 用户名>.github.io/<repository 的名字>` 访问。你需要检查你的 Hexo 配置文件，将 `url` 修改为 `https://<你的 GitHub 用户名>.github.io/<repository 的名字>`、将 `root` 的值修改为 `/<repository 的名字>/`
+1. 开启你在 GitHub 的储存库，并前往 **Settings** 页面。更改你的 **Repository name** 使你的部落格网址变成 <b>username.github.io/_repository_</b>，_repository_ 可以是任何名称，例如 _blog_ 或 _hexo_。
+2. 编辑你的 **\_config.yml**，将 `url:` 更改为 <b>https://_username_.github.io/_repository_</b>。
+3. Commit 并 push 到预设分支上。
+4. 当部属完成后，在 `gh-pages` 分支可以找到产生的网页。
+5. 在 GitHub 储存库中，前往 **Settings** > **Pages** > **Source**，并将 branch 改为 `gh-pages` 后储存。
+6. 前往 _username_.github.io/_repository_ 查看。
 
-## 私有 Repository
+## 一键部属
```
Comment (SukkaW) [inline source/zh-cn/docs/github-pages.md:74]:
```suggestion
## 一键部署
```

File: source/zh-cn/docs/github-pages.md [hunk 3]
```diff
@@ -1,68 +1,100 @@
...
+        run: npm install
+      - name: Build
+        run: npm run build
+      - name: Deploy
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          publish_dir: ./public
 ```
+5. 当部属作业完成后，产生的页面会放在储存库中的 `gh-pages` 分支。
```
Comment (SukkaW) [inline source/zh-cn/docs/github-pages.md:55]:
```suggestion
5. 当部署作业完成后，产生的页面会放在储存库中的 `gh-pages` 分支。
```

**Review by SukkaW — CHANGES_REQUESTED** [review]

**Pull request merged**